spec/article_json/export/apple_news/elements/text_spec.rb

```diff
@@ -45,5 +45,14 @@
       let(:output) { '<strong><em>Foo Bar</em></strong>' }
       it { should eq output }
     end
+
+    context 'when the source element has unsupported tags' do
+      let(:content) do
+        'Normal text<script async>Some Script here</script><select><option>OPtion 1</option>' \
```

Comment:
I would add at least one supported HTML tags such as `<h1>Heading 1</h1>` to ensure that tag is not removed.

Reply:
Right now, it's not conserving the valid HTML tags but I'm thinking that we should, actually. Im going to change it so that it only removes the unsupported tags

Reply:
Does the `inner_html` method also work with nested tags? For example if the input is something like this:

```html
Normal text
<script async>Some Script here</script>
<select><option>OPtion 1</option></select>
<form>form content</form>
<p><em>text</em></p>
<form><a href="#">Link</a></form>
```